xdp-redirect/xdp_redirect_dummy.bpf.c

```diff
@@ -0,0 +1,10 @@
+#include <bpf/vmlinux.h>
+#include <bpf/bpf_helpers.h>
+
+SEC("xdp")
+int xdp_redirect_dummy_prog(struct xdp_md *ctx)
+{
+	return XDP_PASS;
+}
```

Comment:
Instead of creating a whole new object + skeleton, let's just use the dummy program from the XDP dispatcher object, like:

```C
DECLARE_LIBXDP_OPTS(xdp_program_opts, opts, .prog_name = "xdp_pass", .find_filename = "xdp-dispatcher.o");
struct xdp_program *dummy_prog = xdp_program__create(&opts);
```